clients/vscode-hlasmplugin/scripts/syntaxes/hlasm_tmgrammar_template.txt

```diff
@@ -1,15 +1,30 @@
 {
   "$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
-  "name": "IBM HLASM",
-  "scopeName": "source.hlasm",
+  "name": "${grammarName}$",
+  "scopeName": "source.${scope}$",
   "patterns": [
     {
-      "include": "#hlasm_syntax"
+      "include": "#${entryPattern}$"
     }
   ],
   "repository": {
+    "code_block": {
+      "begin": "^${listingOffset}$${codeBlockBegin}$",
+      "end": "^${listingOffset}$.*(?:Relocation Dictionary|Ordinary Symbol and Literal Cross Reference|Macro and Copy Code Source Summary|Dsect Cross Reference|Using Map|General Purpose Register Cross Reference|Diagnostic Cross Reference and Assembler Summary).*Page....\\d(?<=^${listingOffset}$.{120}).*",
+      "patterns": [
+        {
+          "include": "#hlasm_syntax"
+        }
+      ]
+    },
     "hlasm_syntax": {
       "patterns": [
+        {
+          "include": "#pageAnnotation"
+        },
+        {
+          "include": "#asma"
+        },
```

Comment:
Shouldn't these be excluded from the normal grammar? At least teh ASMA pattern seems to be active

Reply:
It is active. I wasn't sure if we would want it in normal grammar or not as it looked nice to me. 

Nevertheless, I can see the argument for removing it.